flowauth/Pipfile

```diff
@@ -27,6 +27,7 @@ structlog = "*"
 simplejson = "*"
 freezegun = "*"
 flask-migrate = "*"
+psycopg = "*"
```

Comment:
Why is this being installed in addition to psycopg2?

Reply:
Temporary postgres test fixture only runs with this version...

Reply:
I see that pytest-postgres requires psycopg3, they also mention it being OK so install both. I guess the question is why is it being added to the main packages in addition to psycopg2? (I'm fine with us moving to psycopg3, but that should be done in a separate PR and also remove 2, right?)